vsphere/internal/virtualdevice/virtual_machine_device_subresource.go.orig

```diff
@@ -0,0 +1,986 @@
+package virtualdevice
```

Comment:
Another `.orig` file... I wonder if you meant to include this?

Reply:
Thanks! Looks like that was the last one.